docs/catalyst_101.md

```diff
@@ -136,6 +136,159 @@ Once the existing members have voted to offer you membership, it’s up to you t
+  - what its status is
+- additional ad-hoc discussion topics
+
+Outside of meetings:
+
+- issues and PRs get ingested to the ZenHub board automatically
+- when issue status changes, move the issue in the ZenHub board (maybe - if not, we get to it in the meetings)
+
+### How we'll do it in GH Projects:
+
+Pretty much the same, but with the Catalyst Backlog project instead of ZenHub.
+
+Sprint planning (Monday, week 1):
+
+- for in-flight (in progress, or in review) items in the old sprint:
+  - discuss status and whether we should roll into the new sprint; tag with new sprint if necessary.
+- for items in backlog:
+  - discuss whether we should pull into the new sprint; tag with new sprint if necessary.
+- ad-hoc discussion topics
+
+Mid sprint checkin (Monday, week 2):
+
+- for each item in the sprint:
+  - discuss status
+- ad-hoc discussion topics
+
+Sprint review (Thursday, week 2):
+
+- for each item in the sprint:
+  - discuss status
+- ad-hoc discussion topics
+
+Outside of sprint meetings:
+
+- Dazhong ingests new issues and PRs to the GH Projects board manually (don't worry, this takes like 5 minutes)
+- Everyone moves issues / linked PRs within the GH project when status changes
+
+### Current experiments
+
+For each experiment:
+
+- what's the goal?
+- what's the actual thing we do?
+- how will we assess success?
+- how do we know that sticking to it will feel better than giving up?
+
+#### Only pull scoped, prioritized tickets into the sprint
+
+Goal: make it easier to decide what to work on by agreeing on priority as a team and providing sufficient context for team members to get started on each item.
+
+Activity:
+
+Use the mid sprint checkin as a parallel work block to pull items into Backlog - aiming for at least 2N items in Backlog, where N is the number of people on the team. All items in Backlog should have scope and a next step:
+
+- scope: when would we consider this issue to be done? what criteria have to be met, or what behavior has to exist?
+- next step: what's the first step to getting this working?
+
+In the sprint review, instead of discussing status for each item in the sprint, prioritize the tickets that are in Backlog according to the following schema:
+
+- "Urgent" -> blocking work on high-priority tickets
+- "High" -> if we do not finish this during the next two weeks, we will not hit our quarterly goals
+- "Medium" -> we'd like to finish this during the next two weeks
+- "Low" -> if there's not other stuff to do, it's worth knocking out
+
+In the sprint planning:
+
+- Pull high priority items from in-flight and backlog into the sprint until it feels like we have enough, but not too much, work.
+  - If a low priority item is in flight, we should bump it for higher-priority work.
+
+Outside of the meetings:
+
+- If a new issue comes up, it goes in the Icebox unless there's a compelling case for it to be Urgent, in which case we talk about it as a team and pull it into the sprint if it's truly Urgent.
```

Comment:
How do issues get out of the Icebox? If all New issues go there by default unless they're Urgent will we ever see them again?

I've imagined having a bi-weekly process for categorizing New issues -- deciding either that they're something we do want to tackle and fleshing them out enough to go into the Project Backlog, or that they're beyond our current planning horizion / not well defined enough and put them into the Icebox for later. (Or deciding that we aren't going to do them at all and closing them if appropriate).

Reply:
I also wanted a bi-weekly process for pulling stuff out of Icebox! As you noted elsewhere, I stuffed that in mid-sprint checkin. But it doesn't have to be then. It's nice to have a week or so between doing this and the actual sprint starting, though, because fleshing issues out can take time and it's not always the best synchronous activity.

I think it'd be *nice* to have a higher cadence process for categorizing New stuff, though - something like "daily, a person looks at all the new things: categorize by team, and assess whether they go in the icebox or if they are urgent enough to shove into the sprint"